docs/docs/adding-a-list-of-markdown-blog-posts.md

```diff
@@ -2,7 +2,50 @@
 title: Adding a List of Markdown Blog Posts
 ---
 
-This is a stub. Help our community expand it.
+Once you have added Markdown pages to your site, you are just one step away from being able to list your posts on a dedicated page.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+### Creating the page
+
+The first step will be to create the page which will display your posts, in `src/pages/`. You can for example use `index.js`.
+
+```js
+import React from 'react';
+
+const IndexPage = ({ data: { allMarkdownRemark: { edges } } }) => {
+  const Posts = edges
+    .filter(edge => !!edge.node.frontmatter.date) // You can filter your posts based on some criteria
+    .map(edge => (
+      <Post key={edge.node.id} post={edge.node.frontmatter} />
+    ));
+
+  return <div>{Posts}</div>;
+};
+
+export default IndexPage;
+```
+
+### Creating the GraphQL query
+
+The only thing left to do is to provide the data to your component with a GraphQL query.
```

Comment:
Make clear this goes at the  bottom of the `index.js` file.

Reply:
Done!